compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEBufferizeToAllocation.cpp

```diff
@@ -63,19 +62,20 @@ static FailureOr<SmallVector<Value>> getOperandsToBufferize(
 }
 
 class AMDAIEBufferizeToAllocationPass
-    : public AMDAIEBufferizeToAllocationBase<AMDAIEBufferizeToAllocationPass> {
+    : public impl::AMDAIEBufferizeToAllocationBase<
```

Comment:
I am not sure, why you need the namespace impl:: , I do not see this any in-tree passes that are also using the newer form e.g https://github.com/openxla/iree/blob/e82cfa99b3dc6447fe926329c12414bce75c0507/compiler/src/iree/compiler/Codegen/LLVMCPU/ExpandF16OpToF32Pass.cpp#L54

Reply:
AFAIK, nothing in the `Codegen/` directory is using the new form. In the new form the `*Base` class is within the `impl` namespace (you can take a look at the `Passes.h.inc` generated).